src/main/java/frc/robot/Robot.java

```diff
@@ -91,7 +91,7 @@ public void robotPeriodic() {
     // block in order for anything in the Command-based framework to work.
     CommandScheduler.getInstance().run();
 
-    LogManager.log();
+    // LogManager.log();
```

Comment:
I get we are logging too much stuff right now but I don't think that we should comment this out on main